### PR TITLE
Forms 7489 cypress test cases for checking repeatability functionality in tabs/accordian/wizard when repeatableinstances can be zero

### PR DESCRIPTION
**Pull request by deepprakash345**
cypress test cases for checking repeatability functionality in tabs/accordian/wizard when repeatableinstances can be zero

### CODE REVIEW

**Review by sakshi-arora1 — APPROVED** [review]

File: ui.tests/test-module/specs/prefill/repeatableprefillwithzerooccurrencefortabaccordionwizard.spec.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    let formContainer = null;
+
+    beforeEach(() => {
+        cy.previewForm(pagePath).then(p => {
+            formContainer = p;
+        })
+    })
+
+    it("test that repeatable instances with zero occurance is not in the dom", () => {
+        const children = formContainer._model._children;
```
Comment (rismehta) [inline ui.tests/test-module/specs/prefill/repeatableprefillwithzerooccurrencefortabaccordionwizard.spec.js:27]: Why are we not making use of public API's ? Ideally, test should make use of public API's.

**Review by rismehta — CHANGES_REQUESTED** [review]
-1 for now

File: ui.tests/test-module/specs/prefill/repeatableprefillwithzerooccurrencefortabaccordionwizard.spec.js (hunk 1 above)
Comment (deepprakash345) [reply ui.tests/test-module/specs/prefill/repeatableprefillwithzerooccurrencefortabaccordionwizard.spec.js]: used getModel()

**Review by rismehta — CHANGES_REQUESTED** [review]
minor comments

**Review by rismehta — APPROVED** [review]
LGTM

**Pull request merged**